test/rpc-server/add-module/add-again/README.md

```diff
@@ -0,0 +1,32 @@
+# Add module with `add-module` endpoint again
+
+This test uses [a-to-f](../../resources/a-to-f/) [definition](../../resources/a-to-f/definition.kore) split into two parts:
+
+When the server starts, a `TEST` module is loaded which contains the following transitions:
+
+```
+      a
+     / \
+    V   V
+    b   c
+        |
+        V
+        d
+```
+
+Then a `NEW` module is added using the `add-module` endpoint [parameters](./params.json),
+adding the following transitions:
+
+
+```
+        d
+        |
+        V
+        e
+        |
+        V
+        f
+```
+
+Expected:
+* Module successfully added even though it has been added already, because the content is exactly the same.
```

Comment:
Probably the module would be added twice then?
How is `add-module/add-again` executed? The [test script](https://github.com/runtimeverification/haskell-backend/blob/32d61ab9da5468c38593a138b079a53fcc0fe158/test/rpc-server/runTests.py) does not mention it.